.nvmrc

```diff
@@ -1 +1 @@
-18.18.0
+20.19.0
```

Comment:
I bumped this one in the root to match the one under ui/ to address another issue with the pre-push hooks (it didn't help but this also isn't wrong)